Dockerfile

```diff
@@ -1,14 +1,16 @@
-FROM python:alpine
+FROM python:3-alpine
 
 WORKDIR /usr/src/app
 
-RUN apk add gcc musl-dev libffi-dev openssl-dev libxml2-dev libxslt-dev file llvm-dev make g++
+RUN apk --no-cache add gcc musl-dev libffi-dev openssl-dev libxml2-dev libxslt-dev file llvm-dev make g++
 
 COPY requirements.txt ./
 
-RUN pip install wheel cryptography gunicorn pymysql
+RUN pip install --no-cache-dir wheel cryptography gunicorn pymysql
 RUN pip install --no-cache-dir -r requirements.txt
 
+RUN apk del gcc musl-dev libffi-dev openssl-dev file llvm-dev make g++
```

Comment:
This does not reduce the final image size because the packages are already downloaded a few layers above and kept in the image. Either combine those 4 RUNs into one or leave this line out for now.

Reply:
Thanks! The image is now just 161MB!